### ISSUE TITLE
Keep partial file on error

### ISSUE THREAD

**Issue by UnoSD** [downthemall/downthemall#209]
DTA! 3.0.8
Firefox 45.8.0
Gentoo Linux

### Expected Behavior
An interrupted download should keep the file so it can be resumed later.

### Actual Behavior
The file is deleted.

### Steps To Reproduce
Download a file from a server that doesn't support multiple fetching/a server that get a transient error during your download. (410 error or others)
The file is gone and you have to start all over again.

**Comment by akehsanz** [comment]
This is happening for me also !! Any updates to resolve that??

**Comment by Infi220** [comment]
I completely agree. This is a major issue and I would love to see it fixed so that DTA pauses the download upon an error instead of deleting the file. 

This issue seems to be a dup of #216.
